recipes/pycalverter/meta.yaml

```diff
@@ -0,0 +1,37 @@
+{% set name = "pyCalverter" %}
+{% set version = "1.6.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 91e26c5d06dfe1495e5b946a724ae691e57ea634ee4a9529a7952b837c041111
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - python
+
+test:
+  imports:
+    - calverter
+
+about:
+  home: https://bitbucket.org/bayazee/pycalverter/overview
+  license: GNU General Public (GPL)
```

Comment:
Please, change it to ```license: GPL-2.0```

Reply:
Changed it to `GPL2` as building with `GPL-2.0` failed

Reply:
Strange, I checked and the short version should be ``GPL-2.0``

Reply:
But that is ok ``GPL2`` then